XML/TestCases/FunctionalTests-DPDK.xml

```diff
@@ -79,6 +79,7 @@
         <AdditionalHWConfig>
             <Networking>SRIOV</Networking>
         </AdditionalHWConfig>
+        <SupportedVmSize>Standard_D*</SupportedVmSize>
```

Comment:
DPDK supports regardless size, right? if that is true, we can not put this tag here.